postgresmodels/attackchains.go

```diff
@@ -0,0 +1,68 @@
+package postgresmodels
+
+import (
+	"gorm.io/gorm"
+)
+
+type AttackChainNode struct {
+	gorm.Model            // ID, CreatedAt, UpdatedAt, DeletedAt - ID is required for linking nodes
+	Name          string  `gorm:"type:varchar(255);not null"`
+	Description   *string `gorm:"type:varchar(255)" `
+	AttackChainID string  `gorm:"primaryKey;type:varchar(255);not null"`
+	CustomerGUID  string  `gorm:"primaryKey;type:varchar(255);not null"`
+}
+
+type AttackChainNodeRelation struct {
+	BaseModel
+	ParentNodeID uint `gorm:"primaryKey; not null"`
```

Comment:
we need something like this: https://gorm.io/docs/belongs_to.html
So I think that instead of using here the nodeID alone, we should put the full summary object.